utility/src/main/java/org/oppia/util/parser/CustomBulletSpan.kt

```diff
@@ -48,37 +48,48 @@ class CustomBulletSpan(context: Context) : LeadingMarginSpan {
     start: Int,
     end: Int,
     first: Boolean,
-    layout: Layout?
+    layout: Layout
   ) {
-    if ((text as Spanned).getSpanStart(this) == start) {
-      val style = paint.style
-      paint.style = Paint.Style.FILL
+    val startCharOfSpan = (text as Spanned).getSpanStart(this)
+    val isFirstCharacter = startCharOfSpan == start
 
-      var yPosition = if (layout != null) {
-        val line = layout.getLineForOffset(start)
-        layout.getLineBaseline(line).toFloat() - bulletRadius * 2f
+    if (isFirstCharacter) {
+      // Depending on the phone, x might always be 0. We need to re-calculate it here.
+      val trueX = 24f + gapWidth * indentation
```

Comment:
Please move this ``24f`` to a dimens constant with a name that corresponds to what it's being used for here. Prefer not hardcoding dimensions in Kotlin, and instead keeping them in resource files where we can specialize them if needed.

Reply:
Done.